Makefile

```diff
@@ -1,11 +1,11 @@
 generate-schema:
-	docker-compose exec php-fpm php phing frontend-api-generate-graphql-schema
+	docker compose exec php-fpm php phing frontend-api-generate-graphql-schema
```

Comment:
there are a few articles in https://docs.shopsys.com/en/14.0/installation/ where the minimal docker and docker compose versions are listed, could you update them, please?